maxscale.cnf.d/example.cnf

```diff
@@ -56,6 +54,13 @@ router=readwritesplit
 servers=server1,server2,server3
 user=maxuser
 passwd=maxpwd
+master_failure_mode=error_on_write
```

Comment:
With 2.2, this will cause connections to remain in read-only mode until the client manually reconnects. Might be better to set it to `master_failure_mode=fail_on_write`.

Reply:
fixed